### PR TITLE
Add Ubuntu (22.04 LTS) build dependencies requirements

### PR DESCRIPTION
**Pull request by brodokk**
Hello, this is not much but could help the next person trying to build on Ubuntu 22.04 LTS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,12 @@ Requirements:
 - [Git](https://git-scm.com)
 - [Rust](https://www.rust-lang.org/)
 
+#### Ubuntu (22.04 LTS) requirements
+
+- libfontconfig1-dev
+- pkg-config
+- libxcb-composite0-dev
+
 Start off by cloning the project with git.
```
Comment (ljoonal) [inline README.md:93]: The build instructions would now fall under the header of the Ubuntu

**Review by l-2-j — CHANGES_REQUESTED** [review]
It'd probably be saner to make a new header between Developement and building for requirements and move them there.

**Comment by brodokk** [comment]
> It'd probably be saner to make a new header between Developement and building for requirements and move them there.

Hum... yeah you right, I have another idea. Le me try with another commit directly

**Comment by brodokk** [comment]
Ok I finally do like you said, my other idea was to put the requirements under the build section but they should be before otherwise it feel not organized

**Review by l-2-j — APPROVED** [review]

**Pull request merged**